src/draco/tools/install_test/CMakeLists.txt

```diff
@@ -46,11 +46,3 @@ target_include_directories(install_check PUBLIC "${DRACO_INCLUDE_DIR}")
 target_link_libraries(install_check "${DRACO_LIBRARY}")
 
 install(TARGETS install_check DESTINATION "${CMAKE_INSTALL_FULL_BINDIR}")
-
-if(BUILD_SHARED_LIBS AND WIN32)
-  # Copy the Draco DLL into the bin dir for Windows: Windows does not really
-  # have a concept of rpath, but it does look in the current directory by
-  # default when a program tries to load a DLL.
-  install(FILES "${DRACO_LIBRARY_DLL}"
-          DESTINATION "${CMAKE_INSTALL_FULL_BINDIR}")
-endif()
```

Comment:
It took me some time to understand the purpose of this install_test, so yes moving dll or injecting its location in PATH is still required while running `install_check` executable.

I believe that all this logic, as well as `CMAKE_PREFIX_PATH` & `CMAKE_INSTALL_RPATH` (`CMAKE_INSTALL_RPATH_USE_LINK_PATH` is far better) should be moved in `test.py`.